src/copyrightinserter.ts

```diff
@@ -191,15 +191,42 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.`]
         return line;
     }
 
+    private formatString(header: string, first_line: string, prefix: string, last_line: string): string {
+        var cur_prefix = first_line
+        var result : string = ""
+        for (const line of header.split("\n")) {
+            const new_line = cur_prefix + line
+            result += new_line.trimRight() + "\n"
+            cur_prefix = prefix
+        }
+        return result + last_line
+    }
+
     private formatHeader(template: (holder: string, year: string) => string, data: CopyrightData, language: vscode.LanguageConfiguration): string | undefined {
         const c = language.comments;
 
         let header = template(data.holder, data.year);
         if (c!.blockComment) {
-            header = `${c!.blockComment[0]}\n${header}\n${c!.blockComment[1]}\n`;
+            // We want to format it as follows:
+            //
+            // <Block Comment Start>\n
+            // <Fist line of copyright>\n
+            // <etc.>\n
+            // <Block Comment End>\n
+            if (c!.blockComment[0] == "/*") {
```

Comment:
use operator `===` instead of `==` to emphasis that it compares string values.

Reply:
done